src/ensembl/src/shared/accordion/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,222 @@
+
+      const instance = wrapper.find(Provider).instance() as Provider;
+
+      expect(
+        instance.state.items.filter((item: Item) => item.expanded === true)
+          .length
+      ).toEqual(1);
+    });
+
+    it("expands a collapsed item when its title is clicked, and doesn't close the others", () => {
+      const wrapper = mountTabpanel();
+
+      wrapper.find(FooTitle).simulate('click');
+      wrapper.find(BarTitle).simulate('click');
+
+      const instance = wrapper.find(Provider).instance() as Provider;
+
+      expect(
+        instance.state.items.filter((item: Item) => item.expanded === true)
+          .length
+      ).toEqual(2);
+    });
+
+    it('collapses an expanded item when its title is clicked', () => {
+      const wrapper = mountTabpanel();
+
+      wrapper.find(FooTitle).simulate('click'); // open
+      wrapper.find(FooTitle).simulate('click'); // close
+
+      const instance = wrapper.find(Provider).instance() as Provider;
+
+      expect(
+        instance.state.items.filter((item: Item) => item.expanded === true)
+          .length
+      ).toEqual(0);
+    });
+  });
+
+  it('does not expanded disabled items on click', () => {
+    const wrapper = mount(
+      <Accordion accordion={false}>
+        <AccordionItem disabled={true}>
+          <AccordionItemTitle className="foo">Foo Title</AccordionItemTitle>
+        </AccordionItem>
+      </Accordion>
+    );
+
+    const instance = wrapper.find(Provider).instance() as Provider;
+
+    wrapper
+      .find('.foo')
+      .first()
+      .simulate('click');
+    expect(
+      instance.state.items.filter((item: Item) => item.expanded === true).length
+    ).toEqual(0);
+  });
+
+  it('pre expanded accordion', () => {
+    const wrapper = mount(
+      <Accordion>
+        <AccordionItem expanded={true}>Fake Child</AccordionItem>
+        <AccordionItem>Fake Child</AccordionItem>
+      </Accordion>
+    );
+
+    const instance = wrapper.find(Provider).instance() as Provider;
+
+    expect(
+      instance.state.items.filter((item: Item) => item.expanded === true).length
+    ).toEqual(1);
+  });
+
+  it('works with multiple pre expanded accordion. Extra expands are just ignored.', () => {
+    const hideBodyClassName = 'HIDE';
+    const wrapper = mount(
+      <Accordion accordion={true}>
+        <AccordionItem expanded={true} hideBodyClassName={hideBodyClassName}>
+          Fake Child
+        </AccordionItem>
+        <AccordionItem expanded={true} hideBodyClassName={hideBodyClassName}>
+          Fake Child
+        </AccordionItem>
+      </Accordion>
+    );
+
+    const instance = wrapper.find(Provider).instance() as Provider;
+
+    expect(
+      instance.state.items.filter((item: Item) => item.expanded).length
+    ).toEqual(1);
+    expect(
+      wrapper.findWhere((item: ReactWrapper) =>
+        item.hasClass(hideBodyClassName)
+      ).length
+    ).toEqual(1);
+  });
+
+  it('pre expanded accordion when accordion is false', () => {
```

Comment:
it('expands multiple accordion items when...')

Reply:
That's done.